wazo_call_logd/plugins/cdr/http.py

```diff
@@ -109,12 +109,33 @@ def _output_csv(data, code, http_headers=None):
     return response
 
 
-class CDRAuthResource(AuthResource):
-    representations = {'text/csv; charset=utf-8': _output_csv}
+def format_cdr_result(result):
+    is_csv = (
+        request.args.get('format') == 'csv'
+        or request.headers.get('Accept') == 'text/csv; charset=utf-8'
```

Comment:
I'm really not sure if this is a good way to check for the accept header. Flask-restful, for example, uses `request.accept_mimetypes.best_match()`: https://github.com/flask-restful/flask-restful/blob/master/flask_restful/__init__.py#L494
What do you think about that?

Reply:
Good point, I wondered if I was right when writing it and eventually forgot about it. Thanks for the link! :)